### DISCUSSION TITLE
register page not working

### DISCUSSION THREAD

**Discussion by chandan4744** [hotspotbilling/phpnuxbill#285] [Q&A]
register page (localhost/phpbill/index.php?_route=register) not working plz.. solve the problems @ibnux 

https://github.com/user-attachments/assets/5e1fce6c-bfb1-4b87-89a4-6f873a2651c4

**Comment by ibnux** [comment]
It works
Just you not following installation guide

**Reply by chandan4744** [reply]
Plz share proper guidance @ibnux

**Reply by ibnux** [reply]
The guide is clear in the wiki. 

If you feel phpnuxbill is bad. Just delete it, its very annoying you always mention me . 

I will block it if you mention again 

![Screenshot_20240829-220212~2.jpg](https://github.com/user-attachments/assets/0c49dbc3-c541-4bae-85d8-7503648e4120)
